specification/compute/resource-manager/Microsoft.Compute/GalleryRP/stable/2024-03-03/gallery.json

```diff
@@ -2773,6 +2805,43 @@
       },
       "description": "The gallery artifact version source."
     },
+    "ExecutedValidation": {
+      "type": "object",
+      "properties": {
+        "type": {
+          "type": "string",
+          "description": "This property specifies the type of image version validation."
+        },
+        "status": {
+          "type": "string",
+          "description": "This property specifies the status of the validationProfile of the image version."
+        },
+        "version": {
+          "type": "string",
+          "description": "This property specifies the valid version of the validation."
+        },
+        "executionTime": {
+          "type": "string",
+          "format": "date-time",
+          "description": "This property specifies the starting timestamp."
+        }
+      },
+      "description": "This is the executed Validation."
+    },
+    "PlatformAttribute": {
+      "type": "object",
+      "properties": {
+        "name": {
+          "type": "string",
+          "description": "This property specifies the name of the platformAttributes. It could be either source_image_publisher or source_image_offer or source_image_sku or source_image_version"
```

Comment:
do we want this to be enum? model-as-string?

Reply:
service has this as string, so would like to keep as is

Reply:
its possible to keep as string but have enum values. See: https://github.com/Azure/azure-rest-api-specs/blob/57c5d2257cfa1b11bbbfae87356558ff4c6d459e/specification/compute/resource-manager/Microsoft.Compute/GalleryRP/stable/2024-03-03/gallery.json#L195